views/content/cli.md

```diff
@@ -252,7 +252,7 @@ Once you executed the line above, all you need to check all of your balances is
 
 #### Learn more
 
-* [Account Types, Gas, and Transactions](https://ethereum-homestead.readthedocs.org/en/latest/contracts-and-transactions/account-types-gas-and-transactions.html)
-* [Developing on Ethereum](https://ethereum-homestead.readthedocs.org/en/latest/developing-on-ethereum/dapp-development.html)
-* [Developer tools](https://ethereum-homestead.readthedocs.org/en/latest/developing-on-ethereum/developer-tools.html)
-* [Solidity Documentation](https://solidity.readthedocs.org/)
+* [Account Types, Gas, and Transactions](https://ethereum.gitbooks.io/frontier-guide/content/contracts_and_transactions_intro.html)
+* [Developing on Ethereum](https://ethereum.gitbooks.io/frontier-guide/content/writing_contract.html)
```

Comment:
These are regressions.   They were pointing to the right content, but through the wrong alias.   The should go to ethdocs.org.